Intersect.Server/Entities/Events/Conditions.cs

```diff
@@ -390,16 +390,9 @@ QuestBase questBase
                 var entities = mapInstance.GetEntities();
                 foreach (var en in entities)
                 {
-                    if (en is Npc npc)
+                    if (en is Npc npc && (!condition.SpecificNpc || npc.Base?.Id == condition.NpcId))
```

Comment:
Not this high up, I just wanted the "or" combined.

Reply:
Done